src/core/GlobalErrors.js

```diff
@@ -7,7 +7,12 @@ getJasmineRequireObj().GlobalErrors = function(j$) {
       var handler = handlers[handlers.length - 1];
 
       if (handler) {
-        handler.apply(null, Array.prototype.slice.call(arguments, 0));
+        // Get error from (message, source, lineno, colno, error)
+        var args = Array.prototype.slice.call(arguments, 0);
+        var error = args.find(function(arg) {
```

Comment:
Unfortunately `find` isn't available on all the environments Jasmine supports (IE comes to mind).

Reply:
It's true. I'm going to push another solution. Instead of looking for the first error instance, let's try using the [documented interface](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror#Syntax). If the browser doesn't send all parameters or the last parameter is no error, do what the previous coded did for compatibility.